<launch>
    <arg name="output" default="screen"/>
    <node pkg="adaptive_ctrl" type="occ_grid.py" name="grid_py" output="$(arg output)"/>
</launch>